src/translations/lang/en.json

```diff
@@ -856,7 +856,7 @@
 		"VERIFY_MESSAGE_SIGNED": "**$address** signed **$msg**",
 		"VIEW_ADDR": "View Address",
 		"VIEW_ONLY_WARNING_TOOLTIP": "Note: You cannot send from a view-only account.",
-		"VISIT_DEFIZAP": "Visit [DeFiZap](https://defizap.com/) to learn more.",
+		"VISIT_DEFIZAP": "[Learn more about MyCrypto's Zapper integration.](https://support.mycrypto.com/how-to/defi/how-to-use-zapper)",
```

Comment:
We want to avoid links in translation files:
- use `getKBHelpArticle` to generate link
- pass the link as an argument to the translation key